src/views/config-v2/LeftSidebar.tsx

```diff
@@ -12,37 +12,36 @@ export const LeftSidebar = ({ pdcInjected }: LeftSidebarProps) => {
 
   return (
     <Stack>
-      <Box flex={1} marginY={10}>
-        <Box height="75px"></Box>
+      <Box flex={1} marginY={1}>
         <Text element="h4">Connect data source</Text>
         <Box paddingTop={2}>
           {headers.map((header, index) => (
             <div key={index} data-testid={`${header.label}-sidebar`}>
-              <InlineField label={`${index + 1}`} className={styles.inlineField} labelWidth={3} grow>
-                <LinkButton
-                  variant="secondary"
-                  fill="text"
-                  onClick={(e) => {
-                    e.preventDefault();
-                    const target = document.getElementById(header.id);
-                    if (target) {
-                      const y = target.getBoundingClientRect().top + window.scrollY - 60;
-                      window.scrollTo({ top: y, behavior: 'smooth' });
-                    }
-                  }}
-                >
-                  <div className={styles.sidebarText}>
-                    <div className={styles.sidebarLabel}>{header.label}</div>
-                    {header.isOptional && (
-                      <div className={styles.sidebarOptional}>
-                        <Text color="secondary" variant="bodySmall">
-                          optional
-                        </Text>
-                      </div>
-                    )}
-                  </div>
-                </LinkButton>
-              </InlineField>
+              <Icon name="circle" size="xs" />
+              <LinkButton
+                style={header.isOptional ? { padding: '5px 15px', height: '50px', width: '225px' } : {}}
+                variant="secondary"
+                fill="text"
+                onClick={(e) => {
+                  e.preventDefault();
+                  const target = document.getElementById(header.id);
+                  if (target) {
+                    const y = target.getBoundingClientRect().top + window.scrollY - 60;
+                    window.scrollTo({ top: y, behavior: 'smooth' });
```

Comment:
Is this the "normal" way to handle scroll to? If so, i think we should pull this out of the html and make a function. I would argue we shouldnt have condisionals here